### PR TITLE
Added postFile method, unit tests, and documentation

### PR DESCRIPTION
**Pull request by Mark-de-Haan**

### CODE REVIEW

**Comment by Mark-de-Haan** [comment]
@fdlk can you review?

File: src/molgenis-api-client.js [hunk 1]
```diff
@@ -82,4 +87,31 @@ const delete_ = (url, options_) => {
...
+ * @example <caption>Example of how the use the postFile method</caption>
+ * // Post a file and handle the response
+ * postFile('/plugin/one-click-importer/upload', myAwesomeFile).then(response => {...}, error => {...})
+ *
+ * @param url
+ * @param file
+ */
+const postFile = (url, file) => {
+  const form = new FormData()
+  form.append('file', file)
```
Comment (fdlk) [inline src/molgenis-api-client.js]: Make the parameter name configurable.

File: package.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
   "license": "LGPL-3.0",
   "dependencies": {
     "babel-runtime": "^6.11.6",
+    "form-data": "^2.3.1",
```
Comment (fdlk) [inline package.json]: Doesn't fetch api support this by default?

File: test/molgenis-api-client.spec.js [hunk 3]
```diff
@@ -120,4 +120,30 @@ describe('Client Api', () => {
       delete_.catch(response => assertEquals(response, 'its an error')).then(done())
     })
   })
+
+  describe('postFile', () => {
+    afterEach(fetchMock.restore)
+
+    it('should return a Job URL when post is successful', done => {
```
Comment (fdlk) [inline test/molgenis-api-client.spec.js]: it( 'should return response body when post is successful'

File: src/molgenis-api-client.js (hunk 1 above)
Comment (fdlk) [reply src/molgenis-api-client.js]: Created #10 , let's try to merge with post method.

**Pull request merged**